spinegeneric/cli/check_data_consistency.py

```diff
@@ -61,5 +63,26 @@ def main():
                     if os.path.exists(jsonSidecarPath) == False:
                         print ("Missing jsonSidecar: " + jsonSidecarPath)
 
+    # Checking participants.tsv contents
+    schema = Schema([
+    Column('participant_id', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
+    Column('sex', [InListValidation(['M', 'F'])]),
+    Column('age', [InRangeValidation(18, 60)]),
+    Column('height', [InRangeValidation(0, 250)]),
+    Column('weight', [InRangeValidation(0, 250)]),
+    Column('date_of_scan', [DateFormatValidation('%Y-%m-%d')]),
+    Column('institution_id', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
+    Column('institution', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
+    Column('manufacturer', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
+    Column('manufacturers_model_name', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
+    Column('receive_coil_name', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
+    Column('software_versions', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
+    Column('researcher', [LeadingWhitespaceValidation(), TrailingWhitespaceValidation()]),
+    ])
+
+    errors = schema.validate(tsv_file)
+    print ('\n Checking the contents of participants.tsv')
+    for error in errors:
+        print(error)
```

Comment:
PEP8
```suggestion
        print(error)
        
        
```

Reply:
There's not space between print and (.

Reply:
fixed in https://github.com/spine-generic/spine-generic/pull/232/commits/c7b32600bb1b74055cf4b901695ec52acdb00203